src/components/Header/index.tsx

```
@@ -0,0 +1,100 @@
+}
+
+const Header = ({ pageType = 'default', notificationCount }: HeaderProps) => {
+    const isAuthPage = pageType === 'login' || pageType === 'signup';
+    
+    // 알림 개수 결정
+    const [localCount] = useState(0);
+    const count = notificationCount !== undefined ? notificationCount : localCount;
+    const isDoubleDigit = count >= 10;
+
+    // 알림 드롭다운 오픈 상태
+    const [dropdownOpen, setDropdownOpen] = useState(false);
```

ESLint

[error] 'dropdownOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)